src/main/java/net/neoforged/neoforge/network/filters/VanillaPacketSplitter.java

```diff
@@ -49,69 +45,70 @@ public class VanillaPacketSplitter {
     private static final byte STATE_FIRST = 1;
     private static final byte STATE_LAST = 2;
 
-    public static void register() {
-        Predicate<String> versionCheck = NetworkRegistry.acceptMissingOr(VERSION);
-        EventNetworkChannel channel = NetworkRegistry.newEventChannel(CHANNEL, () -> VERSION, versionCheck, versionCheck);
-        channel.addListener(VanillaPacketSplitter::onClientPacket);
+    @SubscribeEvent
+    public static void register(final RegisterPayloadHandlerEvent event) {
+        event.registrar(NeoForgeVersion.MOD_ID)
```

Comment:
Because that is how it was.
This PR is not going to change the original core behaviour of other subsystems.
The VPS already worked like this using a SimpleChannel, so I just followed suit. It is not better or worse to register stuff together or split, as such I left it exactly as I found it, structurally.